src/wp-includes/comment-template.php

```diff
@@ -1440,12 +1440,11 @@ function comments_template( $file = '/comments.php', $separate_comments = false
 	$comment_author_url = esc_url( $commenter['comment_author_url'] );
 
 	$comment_args = array(
-		'orderby'                   => 'comment_date_gmt',
-		'order'                     => 'ASC',
-		'status'                    => 'approve',
-		'post_id'                   => $post->ID,
-		'no_found_rows'             => false,
-		'update_comment_meta_cache' => false, // We lazy-load comment meta for performance.
```

Comment:
Q: Do we needs to remove `update_comment_meta_cache` from `get_comment_count()` and `wp_comments_personal_data_exporter()` function?

Reply:
@mukeshpanchal27 No, `get_comment_count` and `wp_comments_personal_data_exporter` are set to false, as comment meta is not need. This is set to false as `wp_queue_comments_for_comment_meta_lazyload` was called. `wp_queue_comments_for_comment_meta_lazyload` is no longer needed.